firedrake/ml/pytorch/fem_operator.py

```diff
@@ -81,9 +81,8 @@ def backward(ctx, grad_output):
         if isinstance(adj_input, Constant) and adj_input.ufl_shape == ():
             # This will later on result in an `AdjFloat` adjoint input instead of a Constant
             adj_input = float(adj_input)
-
         # Compute adjoint model of `F`: delegated to pyadjoint.ReducedFunctional
-        adj_output = F.derivative(adj_input=adj_input)
+        adj_output = F.derivative(adj_input=adj_input, options={"riesz_representation": None})
```

Comment:
Riesz map fails for L2 norm. I do not know what should be `adj_output` in this case. For now, I set as a derivative (`Cofunction`).

Reply:
I'd say that unless the user specifies options={"riesz_representation": None}, the output should be a function